app/views/unattended/provisioning_templates/finish/preseed_default_finish.erb

```diff
@@ -65,6 +65,7 @@ echo 'root:<%= root_pass -%>' | /usr/sbin/chpasswd -e
 <% end -%>
 
 <%= snippet 'preseed_networking_setup' -%>
+<%= snippet 'ansible_provisioning_callback' %>
```

Comment:
This is missing the final dash, that would be nice to have to limit white spaces.
If you want I can add that for you.

Reply:
Fixed!  Sorry about that.  Ha, I just noticed you said you could add that for me after doing it.  Well either way.  =)